setup/_metapackage/VERSION.txt

```diff
@@ -1 +1 @@
-10.0.20180419.0
\ No newline at end of file
+10.0.20180727.0
```

Comment:
Is there a reason why all this `setup/` stuff is in here? Isn't this something that OCA Bot generates automatically?

Reply:
As far as I know, they are autogenerated for https://pypi.org/project/setuptools-odoo/
They seem to be here due to a merge https://github.com/OCA/l10n-finland/pull/16/commits/46277158a44c9f01f0bdc4e469dfd2a716a5f479 from main branch

Reply:
Ok. Could you remove them from this PR?

Reply:
It looks a little hairy. There are some merge commits in there, which could make the whole thing really complicated. Maybe try a reset-"rebase":
1. `git checkout -b just-testing 10.0-l10n_fi_business_code`
2. `git fetch origin`
3. `git reset --soft origin/10.0` (Assuming `origin` is the remote pointing to the OCA repository)
4. `git add the-things-you-want-here` (make sure you don't add any `setup/` stuff if it isn't wanted)
5. `git commit`
6. `git push --force-with-lease`

This should "squash" everything so that you only have one commit with the files you want to have. You can also do steps 4 and 5 several times, if you want to have different commits cover different things.

Reply:
@mlaitinen it looks like your review is maybe the only thing holding this back.